### DISCUSSION TITLE
Server was not listing in teleport web ui

### DISCUSSION THREAD

**Discussion by naikbarath33** [gravitational/teleport#21213] [Q&A]
we are adding teleport server manually  without using the teleport ui and it was listing in teleport UI
after removing teleport from server and try to add again /etc/teleport.yaml file not generating and it was through error as below we are not able to add same instance or node again in teleport .how to resolve this. 

2023-02-03 16:07:11 IST [teleport-installer] Found: Teleport v11.3.1 git:api/v11.3.1-0-gccd6a1b02a go1.19.5
2023-02-03 16:07:11 IST [teleport-installer] Writing Teleport node service config to /etc/teleport.yaml
bash: line 475: LABELS_FLAG[@]: unbound variable

**Comment by mvalo** [comment]
same error here on a few "CentOS Linux release 7.7.1908 (Core)" VM. 

We are installing teleport agent by generating a token with `tctl nodes add` and then launching installation script : 
```
bash -c "$(curl -fsSL https://__TELEPORTDOMAIN__/scripts/__TOKEN__/install-node.sh)"
```

Sometimes (not able to get exact reason why it sometimes work / sometimes not), installation ends like below : 
```
Running transaction
  Installing : teleport-13.2.3-1.x86_64                                                                                                                                                                                                 1/1
  Verifying  : teleport-13.2.3-1.x86_64                                                                                                                                                                                                 1/1

Installed:
  teleport.x86_64 0:13.2.3-1

Complete!
2023-07-25 14:23:59 UTC [teleport-installer] Found: Teleport v13.2.3 git:api/v13.2.3-0-g4785e70 go1.20.6
2023-07-25 14:23:59 UTC [teleport-installer] Writing Teleport node service config to /etc/teleport.yaml
bash: line 477: LABELS_FLAG[@]: unbound variable
```

As a workaround, we modify LABELS='' and add one. Then it never fails.
